_docs/_latest/use/cli/scanner.md

```diff
@@ -47,15 +47,15 @@ When Scanner finds a rule violation, it outputs the data to a Cloud SQL database
 | my-project-1 | firewall_rule | organization/1234567890/project/my-project-1/firewall/494704901029517064/ | 1 | disallow_all_ports | FIREWALL_BLACKLIST_VIOLATION | {u'policy_names': [u'gke-staging-9696e33a-all'], u'recommended_actions': {u'DELETE_FIREWALL_RULES': [u'gke-staging-9696e33a-all']}} |
 | my-project-2 | firewall_rule | organization/1234567890/project/my-project-2/firewall/494704901029517064/ | 1 | disallow_all_ports | FIREWALL_BLACKLIST_VIOLATION | {u'policy_names': [u'gke-canary-west-69bb2963-all'], u'recommended_actions': {u'DELETE_FIREWALL_RULES': [u'gke-canary-west-69bb2963-all']}} |
 
-### Bigquery violation
+### BigQuery violation
 
 {: .table .table-striped}
 | resource_id | resource_type | full_name | rule_index | rule_name | violation_type | violation_data |
 |--------------|---|---|---|---|---|---|
 | my-project-1:testdataset1 | bigquery_dataset | organization/1234567890/project/my-project-1/dataset/my-project-1:testdataset1/dataset_policy/dataset/my-project-1:testdataset1/ | 0 | Search for public datasets | BIGQUERY_VIOLATION | {u'access_user_by_email': u'', u'access_special_group': u'allAuthenticatedUsers', u'access_domain': u'', u'access_group_by_email': u'', u'role': u'READER', u'full_name': u'organization/1234567890/project/my-project-1/dataset/my-project-1:testdataset1/dataset_policy/dataset/my-project-1:testdataset1/', u'dataset_id': u'my-project-1:testdataset1', u'view': {}} |
 | my-project-1:testdataset1 | bigquery_dataset | organization/1234567890/project/my-project-1/dataset/my-project-1:testdataset1/dataset_policy/dataset/my-project-1:testdataset1/ | 0 | Search for datasets accessible by users with gmail.com addresses | BIGQUERY_VIOLATION | {u'access_user_by_email': u'my_test_acc_1@gmail.com', u'access_special_group': u'allAuthenticatedUsers', u'access_domain': u'', u'access_group_by_email': u'', u'role': u'READER', u'full_name': u'organization/1234567890/project/my-project-1/dataset/my-project-1:testdataset1/dataset_policy/dataset/my-project-1:testdataset1/', u'dataset_id': u'my-project-1:testdataset1', u'view': {}} |
 
-### Cloudsql violation
+### Cloud SQL violation
```

Comment:
Same comment above about links to headings: this is now `#cloud-sql-violation`.

Reply:
As above, redirect or persistent anchor?

Reply:
Let's just look for any links that referenced the old anchor and update them to the new one.

`grep -Ri "#cloudsql-violation" *.md *`

Reply:
Resolved.